source/fundamentals/data-formats/custom-types.txt

```diff
@@ -282,157 +276,145 @@ The ``fallback_encoder`` Callable
+   ...         return Decimal128(value)
+   ...     return value
+   ...
 
-After declaring the callback, we must create a type registry and codec options
-with this fallback encoder before it can be used for initializing a collection:
+After declaring the callback, create a type registry and codec options
+with this fallback encoder before initializing a collection:
 
 .. code-block:: python
 
-  >>> type_registry = TypeRegistry(fallback_encoder=fallback_encoder)
-  >>> codec_options = CodecOptions(type_registry=type_registry)
-  >>> collection = db.get_collection("test", codec_options=codec_options)
-  >>> collection.drop()
+   >>> type_registry = TypeRegistry(fallback_encoder=fallback_encoder)
+   >>> codec_options = CodecOptions(type_registry=type_registry)
+   >>> collection = db.get_collection("test", codec_options=codec_options)
+   >>> collection.drop()
 
-We can now seamlessly encode instances of ``~decimal.Decimal``:
+You can now encode instances of ``~decimal.Decimal``:
 
 .. code-block:: python
 
-  >>> collection.insert_one({"num": Decimal("45.321")})
-  InsertOneResult(ObjectId('...'), acknowledged=True)
-  >>> mydoc = collection.find_one()
-  >>> pprint.pprint(mydoc)
-  {'_id': ObjectId('...'), 'num': Decimal128('45.321')}
-
+   >>> collection.insert_one({"num": Decimal("45.321")})
+   InsertOneResult(ObjectId('...'), acknowledged=True)
+   >>> mydoc = collection.find_one()
+   >>> pprint.pprint(mydoc)
+   {'_id': ObjectId('...'), 'num': Decimal128('45.321')}
 
 .. note::
 
-  Fallback encoders are invoked *after* attempts to encode the given value
-  with standard BSON encoders and any configured type encoders have failed.
-  Therefore, in a type registry configured with a type encoder and fallback
-  encoder that both target the same custom type, the behavior specified in
-  the type encoder will prevail.
-
-
-Because fallback encoders don't need to declare the types that they encode
-beforehand, they can be used to support interesting use-cases that cannot be
-serviced by ``TypeEncoder``. One such use-case is described in the next
-section.
-
+   Invoke fallback encoders *after* attempts to encode the particular value
+   with standard BSON encoders and any configured type encoders have failed.
+   Therefore, in a type registry configured with a type encoder and fallback
+   encoder that both target the same custom type, the behavior specified in
+   the type encoder prevails.
 
 Encoding Unknown Types
 ^^^^^^^^^^^^^^^^^^^^^^
 
-In this example, we demonstrate how a fallback encoder can be used to save
-arbitrary objects to the database. We will use the the standard library's
-``pickle`` module to serialize the unknown types and so naturally, this
-approach only works for types that are picklable.
+This example describes how to use a fallback encoder to save
+arbitrary objects to the database. It uses the standard library's
+``pickle`` module to serialize the unknown types, and works only for types that
+are pickleable.
```

Comment:
had to google this, but the [pickle docs](https://docs.python.org/3/library/pickle.html#pickle.PickleBuffer) use "picklable", so i would go with that. (google also suggested "pickleball", which would be a fun addition to the docs.)

```suggestion
are picklable.
```

Reply:
pickleball is preferred